src/gui/tray/ActivityListModel.h

```diff
@@ -74,7 +74,9 @@ class ActivityListModel : public QAbstractListModel
     void removeActivityFromActivityList(int row);
     void removeActivityFromActivityList(Activity activity);
 
-    Q_INVOKABLE void triggerActionAtIndex(int id) const;
+    Q_INVOKABLE void handleClick(int id) const;
```

Comment:
I liked the trigger wording better I think... handle is too passive and click is too much tied to how we use it on the GUI (which can change). Old name was slightly misleading though so what about renaming it `triggerDefaultAction`? And I'd also rename `id` into `activityIndex` to be consistent with the new method you introduced.